app/assets/javascripts/provider/admin/apiconfig/services/integration.js.coffee

```diff
@@ -105,6 +105,16 @@ $(document).on 'initialize', '#proxy', ->
     setUpWarningForPattern rule.find('.pattern')
     false
 
+  $("a[href=\"#delete-backend-api-config\"]").live "click", ->
```

Comment:
I know we're going to ditch this script, but since we are not doing it anytime soon, not asking for js test, but maybe a cuke? if you think it's worthy

Reply:
The deletion of a backend api config is broken in this PR, @didierofrivia. I say we send a separate PR to fix that and add the cuke.